netket/exact.py

```diff
@@ -194,46 +200,20 @@ def steady_state(lindblad, sparse=False, method="ed", rho0=None, **kwargs):
             w, v = eigsh(ldagl, which="SM", k=2)
 
         print("Minimum eigenvalue is: ", w[0])
-        rho = matrix(v[:, 0].reshape((M, M)))
+        rho = array(v[:, 0].reshape((M, M)))
```

Comment:
After rebasing on `master`, `v[:, 0].reshape((M, M))` is always an array.
```suggestion
        rho = v[:, 0].reshape((M, M))
```